packages/components/src/Checkbox/Checkbox.stories.tsx

```diff
@@ -67,13 +68,17 @@ export default meta;
 type Story = StoryObj<typeof meta>;
 
 export const Basic: Story = {
-  render: args => <Checkbox size="small" {...args} />,
+  render: args => <MyCheckbox size="small" {...args} defaultChecked />,
 };
 
 export const WithFieldGroup: Story = {
   render: args => (
     <FieldGroup labelWidth="100px">
-      <Checkbox {...args} />
+      <MyCheckbox {...args} />
     </FieldGroup>
   ),
 };
+
+export const RACCheckbox: Story = {
```

Comment:
They are now all with the RAC Checkbox or is this otherwise than the others? @OsamaAbdellateef

Reply:
yeah you are right , they are all RACCheckBox , except the`CheckboxGroup` it is using the old `Checkbox`